pkg/repository/git.go

```diff
@@ -201,8 +210,8 @@ func (ogr oktetoGitRepository) Head() (*plumbing.Reference, error) {
 	return ogr.repo.Head()
 }
 
-func (ogr oktetoGitRepository) CommitObject(h plumbing.Hash) (gitCommitInterface, error) {
-	return ogr.repo.CommitObject(h)
+func (ogr oktetoGitRepository) Log(o *git.LogOptions) (object.CommitIter, error) {
+	return ogr.repo.Log(o)
```

Comment:
what does this do?

Reply:
We need to get the latest commit for a folder. The dependency that we are using doesn't have a way to do it but has a wrapper for the `git log` command which has the commit of the folder. That's why I used this. 

If you mean when do we have to have this function is to make testable code